src/test/resources/bookmap1/exp/xhtml/index.html

```diff
@@ -8,7 +8,7 @@
 <meta name="DC.rights.owner" content="(C) Copyright 2014" />
 <meta name="generator" content="DITA-OT"/>
 <meta name="DC.type" content="bookmap" />
-<meta name="DC.title" content="Batty Things Caring for your fruit bat How to properly love and care for your bat." />
+<meta name="DC.title" content="Batty ThingsCaring for your fruit batHow to properly love and care for your bat." />
```

Comment:
The old test assumed the nested elements were indented and there was whitespace between them. Preprocess2 doesn't retain ignorable whitespace.